README.md

```diff
@@ -92,17 +92,24 @@ The milestone needs testing on all platforms. Please report issues when found.
     - D3 Open Source compiles level scripts in their own hogfiles. Make sure you copy and overwrite `d3-{platform}.hog`.
 
 ## Building
-#### Building - Windows
-1. Make sure that you have Git and Visual Studio 2022 with the “Desktop development with C++” workload and the “C++ MFC for latest v143 build tools (x86 & x64)” component. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
+### Dependencies
+The build process uses [**CMake**](https://cmake.org/) and, by default, [**Ninja**](https://ninja-build.org/). You must install these; the project cannot locate them for you. The source code depends on [**SDL2**](https://github.com/libsdl-org/SDL) and [**zlib**](https://github.com/madler/zlib). You can supply these dependencies yourself via your system's library management tools, or the build system can locate the dependencies for you using [vcpkg](https://github.com/microsoft/vcpkg), a cross-platform dependency-management system developed by Microsoft. The official builds source their dependencies from vcpkg.
 
+### Installing and using vcpkg
+* When building for Windows, vcpkg is already installed and configured when using the Developer Tools Command Prompt.
```

Comment:
The Command Prompt that we tell users to use doesn’t have the phrase “Developer Tools” in its title. I would reword that sentence to make it more clear that the vcpkg thing applies to all of the “-for VS 2022” Command Prompts:

```suggestion
* When building for Windows, vcpkg is already installed and configured when using any of the Visual Studio Command Prompts.
```